person.rb

```diff
@@ -0,0 +1,54 @@
+class Person
+  def initialize(age:, name: 'Unknown', parent_permission: true)
+    @id = 1
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  #  rubocop: disable Style/TrivialAccessors
+  def id
+    #  rubocop: enable Style/TrivialAccessors
+    @id
+  end
+
+  #  rubocop: disable Style/TrivialAccessors
+  def name
+    #  rubocop: enable Style/TrivialAccessors
+    @name
+  end
+
+  #  rubocop: disable Style/TrivialAccessors
+  def age
+    #  rubocop: enable Style/TrivialAccessors
+    @age
+  end
+
+  #  rubocop: disable Style/TrivialAccessors
+  def name=(name)
+    #  rubocop: enable Style/TrivialAccessors
+    @name = name
+  end
+
+  #  rubocop: disable Style/TrivialAccessors
+  def age=(age)
+    #  rubocop: enable Style/TrivialAccessors
+    @age = age
+  end
+
+  #  rubocop: disable Naming/PredicateName
+  def is_of_age?
+    #  rubocop: enable Naming/PredicateName
+    return true if @age >= 18
+
+    false
+  end
+
+  def can_use_services?
+    return true if is_of_age? || @parent_permission
+
+    false
+  end
+
+  private :is_of_age?
+end
```

Comment:
- [ ] Kindly,  Try to fix rubocop errors without disabling it, this makes your code looks unprofessional and unclear, you can run `rubocop` to check for errors and `rubocop --fix` to fix them. check the screenshot below:

--------rubocop------------

![ruby](https://user-images.githubusercontent.com/55635977/135825782-10c65f6f-6f35-4510-9744-42ae3f900140.png)
--------rubocop --fix ------------

![ruby-1](https://user-images.githubusercontent.com/55635977/135826055-2dec1262-0443-46bd-9619-77fcd287e0e3.png)

Reply:
Done